common/v2/services/WalletService/walletconnect/useWalletConnect.ts

```diff
@@ -86,7 +87,7 @@ export function useWalletConnect() {
       } else {
         return service
           .sendTx(tx)
-          .then((txHash: ITxData) => {
+          .then((txHash: string) => {
```

Comment:
We now have a `ITxSigned` and `ITxHash` `Brand` types and type guards to deal with a response from a wallet. It's a slight improvement over using a simple `string`.
https://github.com/MyCryptoHQ/MyCrypto/blob/85c493da74a334806eb476bbc391074e6566aff3/common/v2/utils/typeGuards.ts#L8

Reply:
Changed to ITxHash, thanks for the info!